app/services/order_sorter.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class OrderSorter < ApplicationService
+  DEFAULT_FILTER = 'waiting_for_processing'
+
+  def initialize(params, carts = Cart.all)
+    @params = params
+    @carts = carts
+  end
+
+  def call
+    current_sort = Cart::AVAILABLE_FILTERS.values.include?(@params[:status]) ? (@params[:status]).to_s : DEFAULT_FILTER
```

Comment:
`(@params[:status]).to_s`
Why to_s? Isn't it already a string?

Reply:
You're right. It already comes as a String.
Removed unneeded ```to_s```